packages/carbon-web-components/src/components/content-switcher/content-switcher-item.ts

```diff
@@ -133,6 +136,7 @@ export default class CDSContentSwitcherItem extends FocusMixin(LitElement) {
       type="button"
       role="tab"
       class="${className}"
+      part="button"
```

Comment:
This is great. There is also a child `<span>` element that needs a `part` name as well. Probably `label` makes sense as a name for that one.

Reply:
Oops, thanks!